qa-tests-backend/src/main/groovy/services/ImageService.groovy

```diff
@@ -1,5 +1,6 @@
 package services
```

Comment:
How about adding `@CompileStatic` for this class?

Reply:
What do we gain from that? Seems unrelated to the retry change.

Reply:
it will check types and compile during building not during runtime. I think it's worth adding whenever we change non test code.